workspaces/src/network/sandbox.rs

```diff
@@ -206,3 +234,106 @@ impl Sandbox {
         Ok(())
     }
 }
+
+#[cfg(feature = "experimental")]
+impl Sandbox {
+    pub(crate) async fn changes_in_block(&self, block_reference: BlockReference) -> Result<()> {
+        // NOTE: RpcStateChangesInBlockByTypeResponse is an empty struct with no fields, so don't do anything with it.
```

Comment:
This is not true: https://docs.rs/near-jsonrpc-client/latest/near_jsonrpc_client/methods/EXPERIMENTAL_changes_in_block/struct.RpcStateChangesInBlockByTypeResponse.html

Reply:
Missed that, I'll update